frontend/app/src/pages/tickets/TicketModalPage.tsx

```diff
@@ -68,8 +72,13 @@ export const TicketModalPage = observer(({ setConnectPerson }: Props) => {
 
   const goBack = async () => {
     setVisible(false);
-    await main.getPeopleBounties({ page: 1, resetPage: true });
-    history.goBack();
+    setisDeleted(false);
+    if (matchPath(location.pathname, { path: '/bounty/:bountyId' })) {
+      await main.getPeopleBounties({ page: 1, resetPage: true });
+      history.push('/bounties');
+    } else {
+      history.goBack();
+    }
```

Comment:
I think this has been changed in this PR not sure if you merged master properly https://github.com/stakwork/sphinx-tribes/pull/1016/files#diff-34bc5760f16f76b08d0d444daf69f9af6cf652502e0de1cbe1d5cb15f41bbccbL71

Reply:
I think it is merged properly still I'll check and update accordingly

Reply:
I think just the `setIsDeleted(false)` should be in there but the code below should not have been modified